app/assets/javascripts/Components/submission_table.jsx

```diff
@@ -192,6 +196,16 @@ class RawSubmissionTable extends React.Component {
     });
   };
 
+  setMarkingStates = (marking_state) => {
```

Comment:
This is nice, but actually can you pull out this change into a separate PR? I'd like to put this into the next minor release separately.

Reply:
#4336